src/server/server.js

```diff
@@ -6,11 +6,30 @@ const cookieParser = require('cookie-parser')
 const config = require('./config')
 const { Issuer } = require('openid-client')
 const { generators } = require('openid-client')
+const { custom } = require('openid-client');
+const tunnel = require('tunnel');
 
 const app = express()
 const port = config.server.port
 
 let azureClient = null
+let proxyAgent = null
+
+if (process.env["HTTP_PROXY"]) {
+    proxyAgent = tunnel.httpsOverHttp({
+        proxy: {
+            host: process.env["HTTP_PROXY"].split(":", 2)[0],
```

Comment:
Burde en gjort en `let httpProxy = process.env["HTTP_PROXY"]` for å slippe å gjenta uttrykket + at koden gir en garanti for at det er samme verdi som brukes alle steder ?

Reply:
Burde vi ha logget en linje også hvis proxy ikke er satt? Kan være greit å ha ifm feilsøking hvis man har glemt å sette den

Reply:
👍 til begge